docs/06-contributors/99-maintainers/050-operations.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Operations Overview
+id: operations
+keywords: [operations]
+---
+
+## Coverage
+
+"Wing" encompases several projects/sites.
+This document covers operations for the following:
+
+- https://docs.winglang.io (https://github.com/winglang/docsite)
+- Wing (https://github.com/winglang/wing)
+- Wing Console (https://github.com/winglang/console)
+
+## Severity Levels
+
+Incidents are classified by severity level.
+The severity level is determined by the impact of the incident on user experience.
+
+| Severity | Description | Example |
+| - | - | - |
+| P0 | Extreme user interruption, no reasonable workaround | Unable to install Wing |
+| P1 | Major degradation of user experience, possible workarounds | Main build is broken in CI (cannot publish) |
```

Comment:
I think we need to distinguish between user facing issues (p1) and contributor facing issues (p2).

By definition user facing issues have higher severity.
 
Broken build is p2